stam.py

```diff
@@ -0,0 +1,20 @@
+import os
+

```

Comment:
Your code doesn't pass a Coding convention called flake8.
In coding, using a well established coding conventions will help others to read your code.
check you comments here: https://github.com/osherdp/python-training/pull/77/checks